source/samples/pubsub/PubSubFeature.h

```diff
@@ -73,16 +77,15 @@ namespace Aws
                     /**
                      * \brief Location of file containing data to publish
                      */
-                    std::string pubFile;
+                    std::string pubFile = DEFAULT_PUBLISH_FILE;
```

Comment:
Think this code should be moved to Config.cpp and Config.h so that we do not override the expanded path. Also by doing this we won't have to call expanded path function multiple times.

Reply:
will address it in a separate PR